### ISSUE TITLE
0.15.2 exception at start under TinyCore

### ISSUE THREAD

**Issue by Aminuxer** [atauenis/webone#87]
// non-critical, need small fix

When start 0.15.2 under tinycore linux this message appears:

```
Unhadled Exception. System.InvalidOperationException:
Cannot read keys when either application does not have a console or when console input has been redirected.
Try Console.Read.
   at System.ConsolePal.ReadKey(Boolean intercept)
   at System.ConsolePal.ReadKey()
   at WebOne.Program.GetConfigurationFileName()
   at WebOne.Program.Main(Srting[] args)

```
This message appear after upgrade 0.15.1 to 0.15.2 under this VM with Linux TinyCore:
https://rutracker.org/forum/viewtopic.php?t=6258575

WebOne starting inside `screen` with loop-script.

Proxy server after this starting and listen 8080 tcp-port correctly, webone worked correctly.

**Comment by atauenis** [comment]
Before this error (seems, it appear over STDERR) there should be a more detailed error in STDOUT, starting with `ERROR: Custom configuration file is not found: `. But I can't find if STDOUT have logged anywhere. By manual running `webone` in console, this error message don't appear.

**Comment by atauenis** [comment]
Seems that fix is working, so v0.15.3 don't crashing.

**Comment by Aminuxer** [comment]
Yes, 0.15.3 work correctly under linux tinycore
